### DISCUSSION TITLE
PV Inverter registers as being on AC Input 1

### DISCUSSION THREAD

**Discussion by rrroonn** [freakent/dbus-mqtt-devices#113] [Q&A]
Hi,

I have a PV system on AC Output from which I extract data using external software and I use MQTT /Node-Red to send to GX using this driver on the Venus device - which works well - thank you.

The problem that I have (maybe just my understanding) is that I don't see a way of registering which AC in/output the PV is on.  So I manually set it on the venus display but that doesn't stick over a reboot.

If I reboot and have my client reregister, the PV shows on the AC Input-1 which screws up data on the VRM as well as the Venus GX display.

This is how I currently register...
![image](https://github.com/freakent/dbus-mqtt-devices/assets/20292325/567c2055-4c2f-4625-b294-99f7dd11b86e)

Is there are way to programatically register the device on AC-Output?

thanks,

Ron

**Comment by freakent** [comment]
Hi Ron, thanks for the feedback and your support of this project. 

I recently found a bug that was causing settings to not be saved correctly. I have fixed that bug in the next release and Settings now correctly survive a reboot. I have the next release ready to go. 

It sounds like I also need to review the pvinverter settings more closely. The pvinverter definitions were community supplied originally. I'll take a look and update this discussion.
